cibyl/utils/time.py

```diff
@@ -15,10 +15,15 @@
 """
 
 
-def as_minutes(ms: int) -> int:
-    """Converts ms to mins.
+def as_minutes(duration: float, unit: str = "ms") -> float:
+    """Converts a duration to mins.
 
-    :param ms: The ms to converts.
+    :param duration: The duration to convert.
+    :param unit: The unit the duration is in.
     :return: The time in minutes.
     """
-    return ms / 60000
+    if unit == "ms":
+        return duration / 60000
+    else:
+        # assume seconds as default unit
+        return duration / 60
```

Comment:
If the unit is unknown, it is better practice to raise a NotImplementedError. The documentation says nothing about the assumption, so it is an unexpected behavior waiting to happen.

Reply:
Good point, I've changed it